guidelines/style-guidelines.md

```diff
@@ -0,0 +1,162 @@
+## Table of contents
+
+1. [How we write](#how-we-write)
+2. [Grammar and punctuation](#grammar-and-punctuation)
+3. [Links](#links)
+4. [Lists](#lists)
+5. [Numbers](#numbers)
+6. [Symbols, currency and abbreviations](#Symbols,-currency-and-abbreviations)
+7. [Common terms](#common-terms)
+
+
+## How we write
+
+#### Voice and tone
+
+MetaBrainz speaks in a friendly, direct and easily understandable manner. We:
+* use plain, simple, language when possible
+* use short sentences
+* use an active voice (avoid the passive tense).
+
+#### Spelling
+
+We use US English.
```

Comment:
I'm not sure this is currently true everywhere at all. Should it be? Is US English more understandable for English speakers outside the West than British?

Reply:
Firstly, thanks for the notes!!

To this point: I don't know what we use. I don't think either is more understandable, it more depends on what we've been using so far. I just put in US because MusicBrainz is technically a US operation (?)

This is to get consistency between:
colour vs color
analyse vs analyze
etc

(Note: none of these guidelines are to *force* people to do these things, just to offer guidance to people who want it)

Currently in MB we seem to use US spelling but I couldn't find that many examples:
https://musicbrainz.org/doc/About/Data_License - license (US)
https://picard-docs.musicbrainz.org/en/config/options_interface_colors.html - color (US)

Reply:
Confirmed US spelling with Rob